app/assets/stylesheets/markdown-editor.scss

```diff
@@ -0,0 +1,39 @@
+.md-editor {
+  .md-footer,
+  .md-header {
+    background: $sidebars-background;
+    display: block;
+    padding: 6px 4px;
+  }
+
+  .md-header {
+    height: 42px;
+    border-bottom: 1px solid $border-grey;
+    margin: 0;
+  }
+
+  .md-preview {
+    background: $white;
+    min-height: 90px;
```

Comment:
Properties should be ordered background, color, height, min-height, overflow, padding, position, width, z-index